frontend/src/worksheets/CreateWorkSheet.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { postWorkSheet } from '../api/WorkSheetAPI';
+import CreateWorkSheetForm from './CreateWorkSheetForm';
+
+function CreateWorkSheet() {
+  const [sent, setSent] = useState(false);
+  const [sentSuccessfully, setSentSuccessfully] = useState(false);
+  const [popUpMessage, setPopUpMessage] = useState('');
+
+  async function postData(values) {
+    try {
+      const response = await postWorkSheet(values);
+      if (response.status === 201) {
+        setPopUpMessage('Munkalap sikeresen létrehozva');
+        setSentSuccessfully(true);
+      } else {
```

Comment:
ha a happy case mindig a 201 és egyéb esetben hibát dob  BE akkor az else-re nincs szükség hiszen a catch majd lekezeli

Reply:
átírtam.